### ISSUE TITLE
Remove isTestFile check?

### ISSUE THREAD

**Issue by kentcdodds** [JoinColony/eslint-import-resolver-jest#37]
I'm talking about this:

https://github.com/JoinColony/eslint-import-resolver-jest/blob/d9f7df54ba68803f2a1a5579004831c1682d4d20/src/index.js#L51-L53

I'm running into a situation where I have a utility file that I use throughout my project's tests, and jest is configured to resolve to that file fine and therefore this plugin doesn't warn about that import.

However, for that utility file, if it tries to import things that jest will resolve, the rule fails because it's not a test file.

For example:

```javascript
// my-util.js
// themes is in `src/themes.js` and jest is configured to resolve `src` via `moduleDirectories`
import themes from 'themes' // <-- error
```

I understand why we'd want to make sure people don't apply this resolver on non-test files, but with this it's giving me a false negative and it's not really reasonable for this plugin to determine whether a certain file could be run within the Jest context.

So instead I propose that we remove the `isTestFile` check and update the config docs to use ESLint's built-in `overrides` feature so people use that to make sure they're only running this setting on the applicable files.

**Comment by chmanie** [comment]
@kentcdodds: I agree, we have a similar situation in some of our projects. Have you noticed any performance implications on this?

**Comment by kentcdodds** [comment]
Nope 😃

**Comment by chmanie** [comment]
Let's remove it then!

**Comment by kentcdodds** [comment]
Sweet. Here's a PR: https://github.com/JoinColony/eslint-import-resolver-jest/pull/38
